activities/Clock.activity/index.html

```diff
@@ -17,7 +17,16 @@
 <script>if (window.module) module = window.module;</script>
 <script data-main="js/loader" src="lib/require.js"></script>
 </head>
-
+<script>
```

Comment:
No JavaScript should be included in the HTML files, it's not supported on some platforms (Chrome OS for example).
You should add an event listener like it's done for other events.